keyboards/keebio/sinc/sinc.c

```diff
@@ -1 +1,53 @@
+/* Copyright 2021 Danny Nguyen <danny@keeb.io>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
 #include "sinc.h"
+#include "split_util.h"
+
+bool led_update_kb(led_t led_state) {
+    // Only update if left half
```

Comment:
Might as well fix at the same time
```suggestion
    if (!led_update_user(led_state)) { return false; }
    // Only update if left half
```